src/buffer-builder.ts

```diff
@@ -182,6 +182,18 @@ export class BufferBuilder {
     return this;
   }
 
+  /**
+   * Register open cash drawer action
+   * @return BufferBuilder
+   */
+  public openCashDrawer(): BufferBuilder {
+    // kick drawer 1, pin 2
+    this.buffer.write(Command.CD_KICK_1);
+    // kick drawer 2, pin 5
+    this.buffer.write(Command.CD_KICK_2);
```

Comment:
`CD_KICK_1` 😬 (can we rename it)

Reply:
It is like 1st cash drawer kick 
and 2nd cash drawer kick